src/Spec.ts

```diff
@@ -562,12 +584,12 @@ export default class Spec {
       (await concatJs(sdoJs, tocJs)) + `\n;let usesMultipage = ${!!this.opts.multipage}`;
     const jsSha = sha(jsContents);
 
+    await this.buildAssets(jsContents, jsSha);
+
     if (this.opts.multipage) {
-      await this.buildMultipage(wrapper, commonEles, jsSha);
+      await this.buildMultipage(wrapper, commonEles);
```

Comment:
To explain the change to do `buildAssets` before `buildMultipage`:

`buildMultipage` makes a copy of the document as it exists when `buildMultipage` is invoked. Previously we were inserting `ecmarkup.js` etc separately for the two kinds of output: we'd add `script src='../ecmarkup.js'` in `buildMultipage`, and then later add `script src='ecmarkup.js'` in `buildAssets`.

Now we put all assets in the single-page version initially and then rewrite those references in the copy in `buildMultipage` like we rewrite any other reference.